### ISSUE TITLE
v2.10.0 WEMO Maker "could not be initialised as Cannot convert undefined or null to object" 

### ISSUE THREAD

**Issue by degofedal** [bwp91/homebridge-platform-wemo#235]
### What issue do you have? Please be as thorough and explicit as possible.
Upgraded from 2.9.1 to the latest version 2.10 and got the following error:
[2/1/2021, 7:28:49 AM] [Wemo] [Garage Door] could not be initialised as Cannot convert undefined or null to object 

### Details of your setup.
Homebridge with UI via Docker

* Which version of Homebridge/HOOBS do you have?
Homebridge v1.2.5

* Which platform do you run Homebridge/HOOBS on (e.g. Raspberry Pi/Windows/HOOBS Box)? Please also mention your version of Node.js/NPM if known.
* Docker on a Synology NAS
* Node.js v14.15.4
* Npm v6.14.10

* Which version of this plugin (homebridge-platform-wemo) do you have? Has the issue started since upgrading from a previous version?
Latest version 2.10.0. The problem startet since upgrading. Rolling back to 2.9.1 solves the issue.

* Which Wemo devices do you have that are causing issues? Please include product models if applicable.
* WEMO Maker with a contact sensor setup as Garage Door

### Please paste any relevant logs below.
<img width="1131" alt="Screen Shot 2021-02-01 at 07 31 42" src="https://user-images.githubusercontent.com/3520690/106424291-38f3b380-6462-11eb-85f6-12d886b7e59d.png">
<img width="419" alt="Screen Shot 2021-02-01 at 07 40 25" src="https://user-images.githubusercontent.com/3520690/106424297-3b560d80-6462-11eb-8de1-5051742d9a6a.png">

```
[2/1/2021, 7:28:49 AM] [Wemo] [Garage Door] could not be initialised as Cannot convert undefined or null to object
```

**Comment by bwp91** [comment]
Hi @degofedal 
I believe I have already fixed this in the latest beta version, could you update for me and confirm?
Thanks

**Comment by degofedal** [comment]
> Hi @degofedal
> I believe I have already fixed this in the latest beta version, could you update for me and confirm?
> Thanks

Hi @bwp91 

Updated to the latest beta and got this error instead:

[2/1/2021, 8:06:47 AM] [Wemo] [Garage Door] could not be initialised as Cannot read property '22XXXXXXXXXX' of undefined

The serial number is present in the config. I tried entering it again and saving the config with same result.

Let me know if there is anything else I can assist with.

**Comment by bwp91** [comment]
Could you try a beta I just pushed? v2.10.1-2
It might take a few minutes for it to become available.
Thanks

**Comment by degofedal** [comment]
> Could you try a beta I just pushed? v2.10.1-2
> It might take a few minutes for it to become available.
> Thanks

Absolutely, this on did the trick. Works as expected 👍

[2/1/2021, 8:33:49 AM] [Wemo] [Garage Door] initialised with s/n 22XXXXXXXXXX and mac address 94:XX:XX:XX:XX:XX.

Thanks so much for your time and also this great plugin. Been using it for the past couple of years! Let me know if you need any other assistance. Happy to help!

**Comment by bwp91** [comment]
This issue has been fixed in v2.11.0!
